src/js/project/CreateProjectWizard.js

```diff
@@ -199,11 +200,8 @@ export default class CreateProjectWizard extends React.Component {
     fetch("/get-project-imagery?projectId=" + projectId)
       .then((response) => (response.ok ? response.json() : Promise.reject(response)))
       .then((data) => {
-        const institutionImageryIds = this.context.institutionImagery.map((i) => i.id);
         this.context.setProjectDetails({
-          projectImageryList: data
-            .map((i) => i.id)
-            .filter((id) => institutionImageryIds.includes(id)),
```

Comment:
I can't understand why we won't need this anymore?

Reply:
The sql function `select_imagery_by_project` already brings the imageries filtered by the institution_id, so we don't need to do the filtering in the front-end